### PR TITLE
update hotkeys with gameWindowId

### PR DESCRIPTION
**Pull request by tthreeoh**
Hotkeys will now only now only respond to the gamewindow as defined per settings.

### CODE REVIEW

File: src/d2r-map.ahk [hunk 1]
```diff
@@ -406,7 +406,6 @@ MapSizeDecrease:
     return
 }
     
-#IfWinActive, ahk_exe D2R.exe
```
Comment (joffreybesos) [inline src/d2r-map.ahk:415]: Why is this removed? Don't we only want these functions to execute when a D2R.exe window is active?

**Review by joffreybesos — COMMENTED** [review]
Looks good!

File: src/d2r-map.ahk (hunk 1 above)
Comment (tthreeoh) [reply src/d2r-map.ahk:415]: They're inherently active through the hotkey. there are some hot keys that are defined under that area, such as tab and space. The problem is is that the #if is not dynamic and is executed before anything else is executed in the script, So even though they're only active when that window is open that code cannot be dynamic. So by defining the hotkeys with the window title,  they will only be active on the client they are assigned to, and not just any client that is active.

**Review by joffreybesos — APPROVED** [review]
